### ISSUE TITLE
It give errror like this ..

### ISSUE THREAD

**Issue by LaxmanDroid** [manvigupta1987/WalkMore#1]
Error:Execution failed for task ':app:processDebugGoogleServices'.
> File google-services.json is missing. The Google Services Plugin cannot function without it. 
   Searched Location: 
  /var/www/html/WalkMore-master/app/src/debug/google-services.json
  /var/www/html/WalkMore-master/app/google-services.json

**Comment by LaxmanDroid** [comment]
Step counting is not occuring

**Comment by manvigupta1987** [comment]
You need to create your own google-services.json file using google Console. Enable Google Fit API to see the step counts. You can use the following link:
https://developers.google.com/fit/android/get-api-key

**Comment by LaxmanDroid** [comment]
okay thanks but it is applicable and stricted to work with only one mail id if ill do enable my google fit api using my gmail. how can i make it dynamic usage with any mail id?

**Comment by LuckyRosy** [comment]
i am new to Android.i'm also facing same problem and i generated goole-services.json file using the link https://developers.google.com/android/guides/google-services-plugin but i dont know how to use resources with that..can you please explain briefly btw when i enable my Google Fit api also the step counting is not occuring.

**Comment by LuckyRosy** [comment]
Hello mam can you please tell me, i want to run this application on my android studio.it gives me errors please help me

**Comment by manvigupta1987** [comment]
What is the error that you are getting ?

**Comment by LuckyRosy** [comment]
im getting those json missing error firstly. then using this the link :https://developers.google.com/android/guides/google-services-plugin i generated json file btw idk how to use that json file within the appliction and i enable my google fit api but the count wasn't occuring.

**Comment by manvigupta1987** [comment]
you need to copy that json file in your project ( as mentioned in the link
above) and then you should debug the code and see why it is not working for
you. Is googleApiClient able to connect ? if yes, onDataPoint listener is
getting called when you move ?

**Comment by LuckyRosy** [comment]
yes i did copied the json file into my folder . btw im new to android mam,  Is
googleApiClient able to connect or not where it is know? and onDataPoint
listener is declared in which class. can you please explain step by step.
